### PR TITLE
Add ApeSzn Portfolio

### PR DESCRIPTION
**Pull request by ADWilkinson**
When submitting a new or existing set details change, make sure you have followed the following steps:

- [x]  Add a new file (or update existing file) into the `set-details` folder that is named as `[insert-your-set-address].setdetails.json` (i.e. 0x1494CA1F11D487c2bBe4543E90080AeBa4BA3C2b.setdetails.json)
- [x] Make sure that the JSON follows the correct schema here: https://github.com/SetProtocol/uniswap-tokenlist/blob/main/set-details.schema.json. You can verify it by pasting the schema into https://www.jsonschemavalidator.net and see if your input JSON matches properly. If you need to have multi-line JSON, you can use a HJSON converter to help with writing it and then paste in the final valid JSON spit out from it (can use https://hjson.github.io/try.html for example)
- [x] If updating existing set details, update the version number appropriately (see the JSON schema above for version numbering explanation).
- [x] If adding brand new token, make sure to add it to the set.tokenlist.json as well following the correct schema from Uniswap: https://github.com/Uniswap/token-lists/blob/master/src/tokenlist.schema.json
- [x] Attach the proper signature (signed current version number [i.e. 1] via TokenSets UI) here (i.e. 0x21fbf0696d5e0aa2ef41a2b4ffb623bcaf070461d61cf7251c74161f82fec3a4370854bc0a34b3ab487c1bc021cd318c734c51ae29374f2beb0e6f2dd49b4bf41c):

0x0fcc3141afec91ffb499df923217b44ff15c5d745a4c63ddee89e4c39b0fc1af0f2397108782e649c521ecefcae10da730e4f5966c5c9edffdb31fd8e8e9f4fd1b

### CODE REVIEW

File: set-details/0x5cb21a4f27eda65263dba6d56f1b9a2406e97e02.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "setDetails": {
+    "chainId": 1,
+    "setName": "ApeSzn Portfolio",
+    "setSymbol": "APE",
+    "setDecimals": 18,
+    "setIcon": "https://raw.githubusercontent.com/SetProtocol/uniswap-tokenlist/main/assets/tokens/apeszn.svg",
+    "setDescription": "The ApeSzn Portfolio is built upon a value based investment thesis and methodology. It aims for full exposure to the cryptocurrency altcoin market, focusing on a maxiumum of 10 simultaneous positions within low capitalisation projects that have significant growth potential. \n\nView the methodology: https://www.pulse.inc/\n\n",
```
Comment (asoong) [inline set-details/0x5cb21a4f27eda65263dba6d56f1b9a2406e97e02.json:8]: Pulse inc?

**Comment by asoong** [comment]
Can you do me a favor and provide the checksum address instead: `0x5CB21A4F27eda65263Dba6d56F1B9A2406e97E02`

File: set-details/0x5cb21a4f27eda65263dba6d56f1b9a2406e97e02.json (hunk 1 above)
Comment (ADWilkinson) [reply set-details/0x5cb21a4f27eda65263dba6d56f1b9a2406e97e02.json]: Fixed, apologies - did this very early in the morning

**Comment by ADWilkinson** [comment]
Is that right @asoong?

**Pull request merged**